tests/bearlib/aspects/conftest.py

```diff
@@ -76,3 +78,13 @@ class SubAspect:
         sour = SubAspect_tastes['sour']
 
     return SubAspect
+
+
+@pytest.fixture
+def aspectsTestBear():
+    class testBear(LocalBear, aspects={
+        'detect': [Metadata.CommitMessage.Shortlog],
+        'fix': [Metadata.CommitMessage.Shortlog.TrailingPeriod]},
+                      languages=['Python']):
```

Comment:
And we should agree on some common syntax style here. Personally, I prefer:

```python
class aspectsTestBear(LocalBear, aspects={
        'detect': [Metadata.CommitMessage.Shortlog],
        'fix': [Metadata.CommitMessage.Shortlog.TrailingPeriod],
}, languages=['Python']):
    ...
```

But all @coala/aspects-developers have to make a common decision about this! :)